test/packer_basic_example_test.go

```diff
@@ -12,7 +12,7 @@ func TestPackerBasicExample(t *testing.T) {
 	t.Parallel()
 
 	// Pick a random AWS region to test in. This helps ensure your code works in all regions.
-	awsRegion := aws.GetRandomRegion(t, nil, nil)
+	awsRegion := "eu-central-1" //aws.GetRandomRegion(t, nil, nil)
```

Comment:
I'm guessing this was just for testing?

If so, you can override the behavior of `aws.GetRandomRegion` without changing any code by using the `TERRATEST_REGION` env var.

Reply:
Yes it was. Only a few regions in the account I'm using has a default VPC